### PR TITLE
Remove qhack banner - DO NOT MERGE UNTIL WEDNESDAY MARCH 1ST

### PR DESCRIPTION
**Pull request by doctorperceptron**

### CODE REVIEW

**Review by Anakhom — APPROVED** [review]
You beat me to it :joy:
LGTM, thank you!

**Review by josh146 — CHANGES_REQUESTED** [review]
Hey @doctorperceptron, something odd has happened in the PR preview 🤔 

image

Everything is 100% width

File: jinja/index.html [hunk 1]
```diff
@@ -7,31 +7,7 @@
...
-                  class="black-text text-center"
-                  target="_blank"
-                  href="https://qhack.ai/"
-                  rel="noreferrer"
-                >
-                  <h5 class="mt-0">Join the Party <i class="fas fa-angle-double-right"></i></h5>
-                </a>
-            </div>
-
+        <div class="pt-5 text-center teal-text text-lighten-2"></div>
```
Comment (Anakhom) [inline jinja/index.html:10]:
```suggestion
        <div class="pt-5 text-center teal-text text-lighten-2">
            <img src="img/pennylane_stacked.png" class="logo" srcset="img/pennylane_logo.svg"/>
        </div>
```

File: jinja/index.html [hunk 2]
```diff
@@ -7,31 +7,7 @@
...
             <img src="img/pennylane_stacked.png" class="logo" srcset="img/pennylane_logo.svg"/>
         </div>
```
Comment (Anakhom) [inline jinja/index.html:12]:
```suggestion

```

**Review by Anakhom — CHANGES_REQUESTED** [review]
Good catch @josh146 ! I think this suggestion ought to fix it.

**Comment by doctorperceptron** [comment]
Thank you! All fixed

image

**Review by josh146 — APPROVED** [review]
Thanks @doctorperceptron!

**Review by AndrewGardhouse — APPROVED** [review]
🚀

**Review by Anakhom — APPROVED** [review]

**Pull request merged**